src/typing_extensions.py

```diff
@@ -2873,3 +2912,43 @@ def __ror__(self, left):
                 if not _is_unionable(left):
                     return NotImplemented
                 return typing.Union[left, self]
+
+
+# Aliases for items that have always been in typing
```

Comment:
Could be good to add a comment here outlining (one reason) why we're not going with the apparently-more-simple approach

```suggestion
# Aliases for items that have always been in typing
# Explicitly assign these (rather than using `from typing import *` at the top),
# so that we get a CI error if one of these is deleted from typing.py
# in a future version of Python
```

Reply:
There's actually a more concrete reason: `ByteString` is in `__all__`, and we don't want it here.